### DISCUSSION TITLE
Regression in Beta projects: cannot open card from Board view

### DISCUSSION THREAD

**Discussion by davidpcaldwell** [community/community#16164] [Projects and Issues] [answered]
Currently, if one clicks on a draft issue from the Board view, a new tab is opened pointing to the Board.

In list view, clicking a draft issue still works.

This has been going on for a couple of days (it worked previously).

**Comment by jeannuel** [comment]
Same here

**Comment by DJj123dj** [comment]
I have also the same problem

**Comment by egost** [comment]
Same here. Duplicate here: https://github.com/github/feedback/discussions/16328

**Reply by egost** [reply]
Recording: https://www.loom.com/share/27384d8508e34b5c92a21c6d479d2310

**Reply by egost** [reply]
Another duplicate: https://github.com/github/feedback/discussions/16201

**Comment by azenMatt** [comment, marked as answer]
This is now fixed, but please let me know if you are still having problems
